# Annotation-of-Klebsiella-pneumoniae

•	Built a hybrid assembly of two Illumina paired-end reads and Oxford Nanopore data, analyzed the resulting alignments of contigs using MaSuRCA and MUMer package, and computed a list of SNPs.

•	Trained my assembly using Glimmer3, a computational gene finder that is based on a Markov model, by annotating the main chromosome. Outputted predicted, translated protein sequences and names into a gtf file.  
